pkg/storage/stores/shipper/bloomshipper/config/config.go

```diff
@@ -8,11 +8,23 @@ import (
 )
 
 type Config struct {
-	WorkingDirectory string `yaml:"working_directory"`
+	WorkingDirectory       string                 `yaml:"working_directory"`
+	BlocksDownloadingQueue DownloadingQueueConfig `yaml:"blocks_downloading_queue"`
+}
+
+type DownloadingQueueConfig struct {
+	WorkersCount              int `yaml:"workers_count"`
+	MaxTasksEnqueuedPerTenant int `yaml:"max_tasks_enqueued_per_tenant"`
+}
+
+func (cfg *DownloadingQueueConfig) RegisterFlagsWithPrefix(prefix string, f *flag.FlagSet) {
+	f.IntVar(&cfg.WorkersCount, prefix+"workers-count", 100, "The count of parallel workers that download Bloom Blocks.")
```

Comment:
I think we should be more conservative regarding the default value. `10` should  be more than sufficient?